transport.go

```diff
@@ -41,3 +47,25 @@ func (m *Manager) Register(s *grpc.Server) {
 func (m *Manager) Transport() raft.Transport {
 	return raftAPI{m}
 }
+
+func (m *Manager) Close() error {
+	m.connectionsMtx.Lock()
+	defer m.connectionsMtx.Unlock()
+
+	err := errCloseErr
+	for _, conn := range m.connections {
+		// Lock conn.mtx to ensure Dial() is complete
+		conn.mtx.Lock()
+		conn.mtx.Unlock()
+		closeErr := conn.clientConn.Close()
+		if closeErr != nil {
+			multierror.Append(err, closeErr)
```

Comment:
You need to use the return value: `err = ...`

Reply:
Updated